subset/network/network_tests.py

```diff
@@ -153,22 +154,22 @@ def test_communication_type_broadcast():
     shell_result = shell_command_with_result(tcpdump_display_all_packets, 0, False)
     all_packets_received = packets_received_count(shell_result)
     if (all_packets_received - broadcast_packets_received) > 0:
-        add_summary("Unicast packets received.\n")
+        add_summary("Unicast packets received. ")
     return 'info'
 
 def test_ntp_support():
     shell_result = shell_command_with_result(tcpdump_display_ntp_packets, 0, False)
     ntp_packets_received = packets_received_count(shell_result)
     if ntp_packets_received > 0:
-        add_summary("NTP packets received.\n")
-        add_packet_info_to_report(ntp_packets_received, packet_request_list)
+        add_summary("NTP packets received. ")
```

Comment:
isn't it easier to include the empty space after every add_summary call in the add_summary function instead? 
so on line 172: 
summary_text += text + " "

Reply:
Done